app/javascript/availability/libraries_locations.json

```diff
@@ -85,10 +87,13 @@
     "CCHCREF-FE": "Fayette - Coal and Coke Heritage Center Reference",
     "CD-AH": "Arts & Humanities, West Pattee, 2nd Floor - CD Collection",
     "CD-HB": "Harrisburg - Compact Disc Collection - 1st Floor",
-    "CD-HN": "Copy unavailable, request via Interlibrary Loan",
+    "CD-HN": "Hazleton - Audio CD",
```

Comment:
Will this location remain in the request_via_ill list? 
https://github.com/psu-libraries/psulib_blacklight/blob/232de3d6ddcb5deaec955a47d0747d695288f88b/app/javascript/availability/libraries_locations.json#L777

Just wanted to point out that, if so changing this label won't have any effect, it will use the label from the ill list. 
It makes sense to change the label to its original though.

Reply:
now moved out of ILL list